src/main/pages/che-7/end-user-guide/ref_devfile-reference.adoc

```diff
@@ -3,6 +3,94 @@
 
 This section contains devfile reference and instructions on how to use the various elements that devfiles consist of.
 
+== Adding projects to a devfile
+
+In most cases devfile contains one or more projects for which development the whole workspace is created.
+Projects are added in `projects` section of devfile.
+
+Each project in a single devfile must have a unique name.
+
+Beside project name each project must have its source specified.
+Project source consists of two mandatory values: `type` and `location`.
+
+`type` defined kind of project source provider.
+ +
+`location` contains URI to where project source is located.
+
+Che support the following project types:
+
+ * `git` - projects which is under git VCS. Location should point to clone link.
+ * `guthub` - the same as above, but for projects hosted on https://github.com/[GitHub] only.
```

Comment:
I guess You wanted to create markdown link:
https://github.com/[GitHub] -> [https://github.com/](GitHub)

Reply:
I used ascii doc format